src/basic/cgroup-util.c

```diff
@@ -357,20 +357,26 @@ int cg_kill(
                 Set *s,
                 cg_kill_log_func_t log_kill,
                 void *userdata) {
-        int r;
+        int r, r2;
```

Comment:
This is not your fault, but could you insert a blank line after the function declaration?